testproject/tests/test_second_step_authentication.py

```diff
@@ -363,63 +355,28 @@ def test_new_method_after_deactivation_same_method_wrong(
         )
     )
     response = client.post(
-        path="/auth/email/deactivate/",
-        data={"new_primary_method": "email"},
-        format="json",
-    )
-
-    assert response.status_code == 400
-    error_code = "new_primary_same_as_old"
-    assert response.data.get("new_primary_method")[0].code == error_code
-
-
-@pytest.mark.django_db
-def test_new_method_after_deactivation_user_doesnt_have_method(
-    active_user_with_many_otp_methods,
-):
-    client = APIClient()
-    active_user, _ = active_user_with_many_otp_methods
-    first_step = login(active_user)
-    first_primary_method = active_user.mfa_methods.filter(
-        is_primary=True,
-    )[0]
-    login_response = client.post(
-        path="/auth/login/code/",
+        path="/auth/sms_twilio/deactivate/",
         data={
-            "ephemeral_token": first_step.data.get("ephemeral_token"),
-            "code": create_otp_code(first_primary_method.secret),
+            "code": handler.create_code(),
         },
         format="json",
     )
-    client.credentials(
-        HTTP_AUTHORIZATION=header_template.format(
-            get_token_from_response(login_response)
-        )
-    )
-    response = client.post(
-        path="/auth/email/deactivate/",
-        data={"new_primary_method": "test"},
-        format="json",
-    )
-
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too